tests/test_observables.py

```diff
@@ -555,6 +566,16 @@ def test_triage_submit(self, *args):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_xforce(self, mock_get=None, mock_post=None):
```

Comment:
we can avoid the duplication of testing code. Check how this is managed in `utils.py`.

You can create a new class `CommonTestCases_ip_url_hash` for this case and handle it similarly.

Reply:
That's so cool! I have changed this now :)